packages/react-strict-dom/src/native/stylex/index.js

```diff
@@ -286,10 +296,7 @@ function processStyle<S: { +[string]: mixed }>(style: S): S {
 
       const maybeLengthUnitValue = CSSLengthUnitValue.parse(styleValue);
       if (maybeLengthUnitValue != null) {
-        result[propName] =
-          maybeLengthUnitValue[1] === 'px'
-            ? maybeLengthUnitValue[0]
```

Comment:
Removing this fixes the unitless lineHeight polyfill. Before, it resulted in eager conversion of values like `24px` to `24`, which would later be interpreted as a unitless value even though the initial value used pixels.